icaruscode/Timing/PMTTimingCorrectionsProvider.h

```diff
@@ -19,32 +19,24 @@
 #include "fhiclcpp/types/Sequence.h"
 #include "cetlib_except/exception.h"
```

Comment:
Once is enough (actually, even too much since the header does not use exceptions, but it's not bad practice to include this since in some sense thrown exceptions are part of the service interface).
```suggestion
```
Also, regrettably we are not using FHiCL validation here, so the `fhiclcpp/types` headers are not needed, while `fhiclcpp/ParameterSet.h` is.
I think `ServiceHandle.h` is also superfluous.